src/core/src/op/slice.cpp

```diff
@@ -123,9 +123,9 @@ bool Slice::evaluate(TensorVector& outputs, const TensorVector& inputs) const {
         shape_infer(this, ov::util::get_tensors_partial_shapes(inputs), make_tensor_accessor(inputs));
     outputs[0].set_shape(output_shapes.front().to_shape());
 
-    const auto starts = ov::get_tensor_data_as<int64_t>(inputs[1]);
-    const auto steps = ov::get_tensor_data_as<int64_t>(inputs[3]);
-    const auto axes = slice_no_axes(this) ? default_axes(starts.size()) : ov::get_tensor_data_as<int64_t>(inputs[4]);
+    const auto& starts = ov::get_tensor_data_as<int64_t>(inputs[1]);
+    const auto& steps = ov::get_tensor_data_as<int64_t>(inputs[3]);
+    const auto& axes = slice_no_axes(this) ? default_axes(starts.size()) : ov::get_tensor_data_as<int64_t>(inputs[4]);
```

Comment:
Why is it needed? Doesn't it break RVO?

Reply:
The `const auto&` will not break RVO. Is reported by Coverity (could be false positive). 
From binary size perspective there is no difference (chance code is same). Leavy as it can silence the Coverity issue and it looks like there is no impact on code.